server/sonar-server/src/main/java/org/sonar/server/computation/step/ParseReportStep.java

```diff
@@ -69,7 +77,7 @@ public void execute(ComputationContext context) {
 
       // and parse!
       int rootComponentRef = reportMetadata.getRootComponentRef();
-      recursivelyProcessComponent(reader, context, rootComponentRef);
+      recursivelyProcessComponent(reader, context, rootComponentRef, defaultAssignee(context.getProjectSettings().getString(DEFAULT_ISSUE_ASSIGNEE)));
```

Comment:
I'm aware of the problem but i want to avoid loading the same information for each component. An alternative is to load the information in the ComputationContext.